### PR TITLE
Fix not loading on Cookie Baker (and add details to error message)

### PR DESCRIPTION
**Pull request by davidgro**
This is a fix for both interpretations of issue #36 - that there is an error message blocking loading (During Business Day), and that the error message isn't descriptive.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,9 +130,20 @@ Then rebuild the assets by running Grunt :
...
+  jA.setAttribute('type', 'text/javascript');
+  jA.setAttribute('src', 'http://localhost:8000/cookie-monster.min.js?' + new Date().getTime());
+  document.body.appendChild(jA);
+}());
+```
+
 Next you can commit your changes and push :
 
-    git commit -am "Edited some stuff"
+    git commit -m "Edited some stuff" src
```
Comment (Anahkiasen) [inline README.md:146]: Why the `src` ?

File: README.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ Game.Init = function() {
 
 Any bug or suggestion should be **opened as an issue** [in the repository](https://github.com/Anahkiasen/cookie-monster/issues) for easier tracking. This allows me to close issues once they're fixed.
 
-Before submitting a bug, make sure to give a shot at the latest version of the plugin on the `develop` branch. For this, simply replace `master` by `develop` in the bookmarklet above.
+Before submitting a bug, make sure to give a shot at the latest version of the plugin on the `develop` branch. For this, simply replace `http://cookie-monster` by `http://dev.cookie-monster` in the bookmarklet above.
```
Comment (Anahkiasen) [inline README.md:101]: Ah yes, good idea to mention that.

File: README.md (hunk 1 above)
Comment (davidgro) [inline README.md:146]: So it doesn't try to make the commit with all the new stuff in dist/ which later would have been in the pull request as well. I made that mistake the first time and had to google how to reset a push.

File: README.md (hunk 2 above)
Comment (davidgro) [inline README.md:101]: Oh. I was trying to make this a separate pull request. I still can't figure out how. and didn't see that it had already merged the second commit into this one

**Comment by davidgro** [comment]
What does "Failed — Scrutinizer: No new issues — Travis: Failed" mean?
What did I do wrong?

**Comment by Anahkiasen** [comment]
Well I had created some tests for Cookie Monster but they were failing before that PR so don't mind it.

File: README.md (hunk 1 above)
Comment (Anahkiasen) [inline README.md:146]: Well actually I just thought of it but yeah you do need to commit the recompiled files, otherwise the file that'll be used for the bookmarklet won't be up to date.

**Comment by DosParkers** [comment]
Tried both cookie monster and dev.cookie monster and both just freeze the game.

File: README.md [hunk 3]
```diff
@@ -130,9 +130,20 @@ Then rebuild the assets by running Grunt :
     ./node_modules/.bin/bower install
     ./node_modules/.bin/grunt
 
+Test your changes by hosting a local http server with the contents of the dist directory and using a modified bookmarklet such as
```
Comment (royaldark) [inline README.md:133]: Would it make sense to make the http server part of the build process? Use [grunt-contrib-connect](https://github.com/gruntjs/grunt-contrib-connect) to host a local server and [grunt-contrib-watch](https://github.com/gruntjs/grunt-contrib-watch) to watch source files and automatically recompile on change.

Comment (Anahkiasen) [inline README.md:133]: Watch is already pulled in theory.

**Comment by Qhorin** [comment]
Is this pull request going to be merged? The README addition about `gem install compass sass-globbing --pre` is helpful. I spent a lot of time on that problem.

Is line 146 the only thing holding up the merge?

**Comment by Anahkiasen** [comment]
I'll merge this locally and fix it myself.

File: README.md (hunk 1 above)
Comment (davidgro) [inline README.md:146]: Sorry, got distracted and gave up on Cookie Clicker because of the freezing thing.
Anyway, I thought of a huge problem with including dist/ ... What about multiple concurrent pull requests? I think the only reasonable solution is to rebuild it yourself after accepting each one.
